src/glib/mine/signalproc.cpp

```diff
@@ -1241,6 +1241,298 @@ PJsonVal TOnlineHistogram::SaveJson() const {
 	return Result;
 }
 
+void TTDigest::Update(const TFlt& V) {
```

Comment:
TFlt vs double
- we use native double for function arguments and declaring local variables
- we only use wrapper TFlt for class members (so they get initialized to default values)